lib/utils.js

```
@@ -4,3 +4,21 @@ import { twMerge } from "tailwind-merge"
 export function cn(...inputs) {
   return twMerge(clsx(inputs))
 }
+
+export function getExperienceDuration(startDate) {
+  const now = new Date();
+  const years = now.getFullYear() - startDate.getFullYear();
+  const months = now.getMonth() - startDate.getMonth();
+
+  let totalMonths = years * 12 + months;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'totalMonths' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)